### DISCUSSION TITLE
Long press duration

### DISCUSSION THREAD

**Discussion by panawel** [mobile-dev-inc/Maestro#1687] [Ideas]
First, thank you so much for that genius tool! Keep up!

I would like to have the ability to choose how 'long press' duration will be. I want to long press an element for 7 seconds for example.
Like that:
- Longpress:
      id:
      Duration: 7000

Tnx so so much!

**Comment by bartekpacia** [comment]
duplicate of https://github.com/mobile-dev-inc/maestro/issues/1663
